src/main/java/com/yelp/nrtsearch/server/field/VectorFieldDef.java

```diff
@@ -243,6 +245,15 @@ private static KnnVectorsFormat getHnswBinaryQuantizedVectorsFormat(
         m, efConstruction, mergeWorkers, executorService);
   }
 
+  @VisibleForTesting
+  static KnnSearchStrategy createKnnSearchStrategy(KnnQuery.FilterStrategy filterStrategy) {
+    return switch (filterStrategy) {
+      case DEFAULT -> KnnSearchStrategy.Hnsw.DEFAULT;
+      case ACORN -> new KnnSearchStrategy.Hnsw(60);
```

Comment:
`KnnSearchStrategy.Hnsw.DEFAULT` is basically `new KnnSearchStrategy.Hnsw(60)`:
https://github.com/apache/lucene/blob/main/lucene/core/src/java/org/apache/lucene/search/knn/KnnSearchStrategy.java#L30C27-L30C60

Is this expected?

Reply:
It is actually 0 in 10.2.2 https://github.com/apache/lucene/blob/releases/lucene/10.2.2/lucene/core/src/java/org/apache/lucene/search/knn/KnnSearchStrategy.java#L30 Given that they are changing the default, I will make it explicitly 0.